packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -507,6 +507,7 @@ function doMovePoint(
             return {
                 ...state,
                 hasBeenInteractedWith: true,
+                focusedPointIndex: action.index,
```

Comment:
This ensures that our reducer is aware of the current focused point for future interactions, such as deleting points.